pages/user/[username]/repos/[repo]/collaborators.js

```diff
@@ -61,6 +69,76 @@ const collaboratorsStyles = theme => ({
   }
 });
 
+function _ListOfCollaborators({ collaborator, removeCollaborator, classes }) {
```

Comment:
This is not a list of collaborator. This is just a single Collaborator.-

Reply:
done